backend/templates/emails/renewal_reminder.html

```diff
@@ -0,0 +1,96 @@
+{% extends 'emails/base.html' %}
+
+{% block content %}
+<h2>Reminder to Renew <b>{{ name }}</b> on Penn Clubs & SAC Fair Signup by 8/24!</h2>
+<p style="font-size: 1.2em">
+    Dear <b>{{ name }}</b> leaders,
+</p>
+
+<p style="font-size: 1.2em; color: #8B2222">
+    You are receiving this notification because you were the contact for your organization during the 2019 SAC
+    Activities Fair. You have not yet renewed your club for the {{ year }} school year.
+    <b>
+        If you do not submit a renewal request by August 24th, your club may not appear on the Penn Clubs
+        website in time for new student orientation.
+    </b>
+    Use the green button below to renew your club. The process takes less than 5 minutes.
+</p>
+
+<p style="font-size: 1.2em">
+    If you have already graduated, <b>PLEASE FORWARD THIS EMAIL</b> to the current leadership of
+    the group so that they may complete the registration process and sign up for the SAC Fair. If
+    they are not currently listed as an officer for your club on Penn Clubs, they will need to
+    email <b>contact@pennclubs.com</b> with their <b>name, PennKey, and club name</b>.
```

Comment:
It turns out that the only information we need is the club name and the email (which usually automatic since they send with their Penn email).

We could consider just saying "send an email from your Penn email with a list of club(s) that you want to renew and your position in those club(s)."

Reply:
Thanks for the detailed comments! I'm happy to keep editing these :)